view/js/main.js

```diff
@@ -166,7 +166,7 @@ $(function() {
 
 	/* event from comment textarea button popups */
 	/* insert returned bbcode at cursor position or replace selected text */
-	$("body").on("fbrowser.image.comment", function(e, filename, bbcode, id) {
+	$("body").on("fbrowser.photos.comment", function(e, filename, bbcode, id) {
```

Comment:
Again, while we're at it.
```suggestion
	$('body').on('fbrowser.photos.comment', function(e, filename, bbcode, id) {
```